main.py

```diff
@@ -132,6 +132,26 @@ def handle_listings():
             abort(500, "Failed to get listings")
         return jsonify(listings)
 
+@app.route('/listings/delete/<listing>', methods=['DELETE'])
+@login_required
+def delete_listing(listing):
```

Comment:
small nit, can you change listing to listing_id just to clarify that this is the Id not the object itself

Reply:
Changed